CONTRIBUTING.md

```diff
@@ -1,71 +1,73 @@
+1. The command checks that running OCamlFormat on its own source code yields an identical result. Run `make fixpoint` for this check only. If OCamlFormat is not integrated with your editor, use `make fmt` to reformat all OCamlFormat source files.
+2. The command ensures the test suite passes. No unexpected diffs should appear. If any, accept all diffs at once with `make regtests-promote`.
 
-The second part of `make test` is to ensure the test suite passes. You should not see any unexpected diff. One can accept all diffs at once with `make regtests-promote`.
+Once `make test` passes, test your pull requests on code in external repositories by executing `tools/test_branch.sh <rev>`, where `<rev>` is the git revision/branch with your changes. If your pull request modifies an option, also run `OCAMLFORMAT=<option>=<value> tools/test_branch.sh <rev>` to test with the option enabled.
 
-Once `make test` passes, pull requests should be tested on the code in a set of external repositories. This can be done by executing `tools/test_branch.sh <rev>` where `<rev>` is the git revision/branch containing the pull request's changes. If a pull request affects an option, `OCAMLFORMAT=<option>=<value> tools/test_branch.sh <rev>` should also be run to test with the option enabled.
+The `tools/test_branch.sh` script compares two `ocamlformat` versions (the specified branch or `HEAD` if omitted, and its main merge base) on the test code and reports any formatting differences. Any formatting errors are shown on the terminal and must be fixed before merging a pull request. Inspect any differences to confirm they are intentional. Pull requests modifying code formatting should generally introduce an option for the alternate style (if uncertain, open an issue for discussion). Differences in the formatted code with the option disabled should be summarized and explained in the pull request discussion.
 
-The `tools/test_branch.sh` script runs two versions of `ocamlformat` (the specified branch, or `HEAD` if omitted, and its merge base with main) on the test code and reports the differences in the formatted code. Formatting failures may also be reported on the terminal, these will need to be fixed before a pull request can be merged. The differences should be inspected to ensure they are as intended. Pull requests changing the format of code should generally introduce an option to enable the alternate style (if in doubt, open an issue for discussion). Any differences in the formatted code with the option disabled should be summarized and explained in the pull request discussion.
+For bash shell users, add the following line to your `~/.bashrc` file for git branch name autocompletion in the `
```

Comment:
This part can be removed as well as `ocamlformat_test_branch`, which are outdated. I don't believe it has any user so no need to update it to the more complex test_branch.